### PR TITLE
EIP-3670 State Tests

### PR DESCRIPTION
**Pull request by hugo-dc**
This PR contains Updated EIP-3670 State tests according to latest spec and container format.

State Tests consists in the following cathegories:

- CREATE/CREATE2/CreateTransaction 
    - Test All valid opcodes are deployed correctly  in EOF Contracts
    - Test All invalid opcodes are not deployed in EOF contracts
    - Test Invalid EOF contracts are not deployed, the following  invalid contracts are considered:
        -  Containing truncated push data at the end
        -  Containing undefined instruction after terminating instruction (STOP)

### CODE REVIEW

File: src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CREATE2_EOF1InvalidOpcodesFiller.yml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      - '0x2f'
+      - '0x49'
+      - '0x4a'
+      - '0x4b'
+      - '0x4c'
+      - '0x4d'
+      - '0x4e'
+      - '0x4f'
+      - '0x56' # JUMP deprecated in EIP-4750
+      - '0x57' # JUMPI deprecated in EIP-4750
```
Comment (shemnon) [inline src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CREATE2_EOF1InvalidOpcodesFiller.yml:69]:
```suggestion
      - '0x57' # JUMPI deprecated in EIP-4750
      - '0x58' # PC deprecated in EIP-4750
 ```

File: src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CREATE_EOF1InvalidOpcodesFiller.yml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      - '0x2f'
+      - '0x49'
+      - '0x4a'
+      - '0x4b'
+      - '0x4c'
+      - '0x4d'
+      - '0x4e'
+      - '0x4f'
+      - '0x56' # JUMP deprecated in EIP-4750
+      - '0x57' # JUMPI deprecated in EIP-4750
```
Comment (shemnon) [inline src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CREATE_EOF1InvalidOpcodesFiller.yml:69]:
```suggestion
      - '0x57' # JUMPI deprecated in EIP-4750
      - '0x58' # PC deprecated in EIP-4750
```

File: src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CreateTransactionEOF1InvalidOpcodesFiller.yml [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+     - 'label inv_2f :raw 0x74ef0001010004020001000203000000000000002f006000526015600bf3'
+     - 'label inv_49 :raw 0x74ef00010100040200010002030000000000000049006000526015600bf3'
+     - 'label inv_4a :raw 0x74ef0001010004020001000203000000000000004a006000526015600bf3'
+     - 'label inv_4b :raw 0x74ef0001010004020001000203000000000000004b006000526015600bf3'
+     - 'label inv_4c :raw 0x74ef0001010004020001000203000000000000004c006000526015600bf3'
+     - 'label inv_4d :raw 0x74ef0001010004020001000203000000000000004d006000526015600bf3'
+     - 'label inv_4e :raw 0x74ef0001010004020001000203000000000000004e006000526015600bf3'
+     - 'label inv_4f :raw 0x74ef0001010004020001000203000000000000004f006000526015600bf3'
+     - 'label inv_56 :raw 0x74ef00010100040200010002030000000000000056006000526015600bf3'
+     - 'label inv_57 :raw 0x74ef00010100040200010002030000000000000057006000526015600bf3'
```
Comment (shemnon) [inline src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CreateTransactionEOF1InvalidOpcodesFiller.yml:56]:
```suggestion
     - 'label inv_57 :raw 0x74ef00010100040200010002030000000000000057006000526015600bf3'
     - 'label inv_58 :raw 0x74ef00010100040200010002030000000000000058006000526015600bf3'
```

File: src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CREATE2_EOF1InvalidOpcodesFiller.yml (hunk 1 above)
Comment (gumb0) [reply src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CREATE2_EOF1InvalidOpcodesFiller.yml:69]: Fixed here and in other files.

**Pull request merged**